configure.ac

```diff
@@ -1,10 +1,10 @@
 AC_INIT([eucalyptus-service-image], [1])
 
-INSTALL_TREE=http://linux.mirrors.es.net/centos/6/os/x86_64/
+INSTALL_TREE=http://linux.mirrors.es.net/centos/7/os/x86_64/
 BASE_MIRROR=${INSTALL_TREE}
-UPDATES_MIRROR=http://linux.mirrors.es.net/centos/6/updates/x86_64/
-EPEL_MIRROR=http://linux.mirrors.es.net/epel/6/x86_64/
-EUCALYPTUS_MIRROR=http://downloads.eucalyptus.com/software/eucalyptus/4.2/rhel/6/x86_64/
+UPDATES_MIRROR=http://linux.mirrors.es.net/centos/7/updates/x86_64/
+EPEL_MIRROR=http://linux.mirrors.es.net/epel/7/x86_64/
+EUCALYPTUS_MIRROR=http://packages.release.eucalyptus-systems.com/yum/tags/eucalyptus-devel/rhel/7/x86_64/
```

Comment:
Don't point the defaults at internal locations; we want them to work when run from anywhere once the software is released.  The latter part is the kicker -- if we don't use the right default now we will forget when release time rolls around just like we always seem to for eucalyptus/clc/Makefile.